<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectClass</class>
 <widget class="QMainWindow" name="ProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,1">
        <item>
         <widget class="Area1" name="area1" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox1">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,7">
                <item>
                 <widget class="QLabel" name="label_title1">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>单变量预测</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pushButton">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>指数平滑法</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>季节指数法</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>ARMA法</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox2">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,7">
                <item>
                 <widget class="QLabel" name="label_title2">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>因素预测</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="sizeConstraint">
                   <enum>QLayout::SetDefaultConstraint</enum>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pushButton_4">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>BP神经网络</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_5">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>小波神经网络</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_6">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>GARCA模型</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox3">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,7">
                <item>
                 <widget class="QLabel" name="label_title3">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>组合预测法</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pushButton_7">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>BP+指数平滑</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_8">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>小波+季节指数平滑</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_9">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>GARCA+指数平滑</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="label_1">
                      <property name="text">
                       <string>下一月</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::PlainText</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label1_data">
                      <property name="text">
                       <string>50</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::AutoText</enum>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>下二月</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::PlainText</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label2_data">
                      <property name="text">
                       <string>300</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::PlainText</enum>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>下三月</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::PlainText</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label3_data">
                      <property name="text">
                       <string>400</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::PlainText</enum>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_5">
                  <property name="title">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string/>
                  </property>
                  <widget class="QLabel" name="label_8">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>180</y>
                     <width>72</width>
                     <height>15</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>专家经验</string>
                   </property>
                  </widget>
                  <widget class="QWidget" name="layoutWidget_2">
                   <property name="geometry">
                    <rect>
                     <x>20</x>
                     <y>10</y>
                     <width>26</width>
                     <height>161</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string>10</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="verticalSlider_4">
                      <property name="toolTip">
                       <string/>
                      </property>
                      <property name="minimum">
                       <number>-10</number>
                      </property>
                      <property name="maximum">
                       <number>10</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="invertedAppearance">
                       <bool>false</bool>
                      </property>
                      <property name="tickPosition">
                       <enum>QSlider::NoTicks</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>-10</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="title">
                   <string/>
                  </property>
                  <widget class="QLabel" name="label_11">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>180</y>
                     <width>72</width>
                     <height>15</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>突发因素</string>
                   </property>
                  </widget>
                  <widget class="QWidget" name="layoutWidget_3">
                   <property name="geometry">
                    <rect>
                     <x>20</x>
                     <y>10</y>
                     <width>31</width>
                     <height>161</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <widget class="QLabel" name="label_14">
                      <property name="text">
                       <string>10</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="verticalSlider_6">
                      <property name="toolTip">
                       <string/>
                      </property>
                      <property name="minimum">
                       <number>-10</number>
                      </property>
                      <property name="maximum">
                       <number>10</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="invertedAppearance">
                       <bool>false</bool>
                      </property>
                      <property name="tickPosition">
                       <enum>QSlider::NoTicks</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_15">
                      <property name="text">
                       <string>-10</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="title">
                   <string/>
                  </property>
                  <widget class="QLabel" name="label_16">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>180</y>
                     <width>72</width>
                     <height>15</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>心理因素</string>
                   </property>
                  </widget>
                  <widget class="QWidget" name="layoutWidget_4">
                   <property name="geometry">
                    <rect>
                     <x>30</x>
                     <y>10</y>
                     <width>26</width>
                     <height>161</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                     <widget class="QLabel" name="label_17">
                      <property name="text">
                       <string>10</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="verticalSlider_7">
                      <property name="toolTip">
                       <string/>
                      </property>
                      <property name="minimum">
                       <number>-10</number>
                      </property>
                      <property name="maximum">
                       <number>10</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="invertedAppearance">
                       <bool>false</bool>
                      </property>
                      <property name="tickPosition">
                       <enum>QSlider::NoTicks</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_18">
                      <property name="text">
                       <string>-10</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="Area2" name="area2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="Area3" name="area3" native="true"/>
        </item>
        <item>
         <widget class="Area4" name="area4" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>单变量预测</string>
    </property>
    <addaction name="action11"/>
    <addaction name="action12"/>
    <addaction name="action13"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>因素预测</string>
    </property>
    <addaction name="action21"/>
    <addaction name="action22"/>
    <addaction name="action23"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>组合预测法</string>
    </property>
    <addaction name="action31"/>
    <addaction name="action32"/>
    <addaction name="action33"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
   </widget>
   <addaction name="menu_4"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action11">
   <property name="text">
    <string>指数平滑法</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>季节指数法</string>
   </property>
  </action>
  <action name="action13">
   <property name="text">
    <string>ARMA法</string>
   </property>
  </action>
  <action name="action21">
   <property name="text">
    <string>BP神经网络</string>
   </property>
  </action>
  <action name="action22">
   <property name="text">
    <string>小波神经网络</string>
   </property>
  </action>
  <action name="action23">
   <property name="text">
    <string>GARCA模型</string>
   </property>
  </action>
  <action name="action31">
   <property name="text">
    <string>BP+指数平滑</string>
   </property>
  </action>
  <action name="action32">
   <property name="text">
    <string>小波+季节指数平滑</string>
   </property>
  </action>
  <action name="action33">
   <property name="text">
    <string>GARCA + 指数平滑</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Area2</class>
   <extends>QWidget</extends>
   <header>area2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Area1</class>
   <extends>QWidget</extends>
   <header>area1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Area3</class>
   <extends>QWidget</extends>
   <header>area3.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Area4</class>
   <extends>QWidget</extends>
   <header>area4.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="project.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>openBox()</slot>
 </slots>
</ui>
